.github/autobuild/get_build_vars.py

```diff
@@ -43,8 +43,9 @@ def get_build_version(jamulus_pro_version):
 
 def set_github_variable(varname, varval):
     print("{}='{}'".format(varname, varval))  # console output
-    print("::set-output name={}::{}".format(varname, varval))
-
+    output = os.getenv('GITHUB_OUTPUT')
+    with open(output, "a") as ghout:
+        ghout.write(f"{varname}={varval}")
```

Comment:
I think this needs a line break, otherwise it fails as soon as a second value is written.

```suggestion
        ghout.write(f"{varname}={varval}\n")
```